### PR TITLE
Address review comments for 2.17.1 cherrypick PR

### PR DESCRIPTION
**Pull request by zeusongit**
### Purpose

Addressing review comments for:
- #13727 

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers

@mjkkirschner

### CODE REVIEW

File: src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs [hunk 1]
```diff
@@ -627,10 +627,9 @@ public DelegateCommand ToggleIsVisibleGroupCommand
...
             {
                 if (toggleIsVisibleGroupCommand == null)
-                    toggleIsVisibleGroupCommand =
-                        new DelegateCommand(ToggleIsVisibleGroup, CanToggleIsVisibleGroup);
-                new DelegateCommand(RemoveGroupFromGroup, CanUngroupGroup);
-
+                {
+                    toggleIsVisibleGroupCommand = new DelegateCommand(ToggleIsVisibleGroup, CanToggleIsVisibleGroup);
+                }
                 return toggleIsVisibleGroupCommand;
```
Comment (zeusongit) [inline src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs:633]: The backing variable is to not initiate the delegate command for every call.

**Review by mjkkirschner — APPROVED** [review]

**Comment by mjkkirschner** [comment]
@zeusongit you meant to make this PR to master and then you will cherry pick it to 2.17.1?

**Comment by zeusongit** [comment]
> 

yes

**Comment by zeusongit** [comment]
> @zeusongit you meant to make this PR to master and then you will cherry pick it to 2.17.1?

yes

**Pull request merged**